dm/master/scheduler/scheduler_test.go

```diff
@@ -1224,3 +1226,41 @@ func (t *testScheduler) TestStartStopSource(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(workers, HasLen, 0)
 }
+
+func checkAllWorkersClosed(c *C, s *Scheduler, closed bool) {
+	for _, worker := range s.workers {
```

Comment:
please check `s.workers` is not empty

Reply:
add length check in https://github.com/pingcap/dm/pull/1639/commits/91eff928a9d8e986c1a6ce3c81299c9d4a98c238